docs/specifications/tests/Basic-TP/basic04.md

```diff
@@ -23,46 +37,97 @@ report problems found in the following areas:
 * Name Server responding with unexpected RCODE (any except "NOERROR")
```

Comment:
RCODE -> [RCODE Name] (with link to IANA RCODE Name page)
"NOERROR" -> "NoError"

Reply:
This also applies in the Test Procedure below where I can see two other occurrences

Reply:
Fixed three places plus added link in link section.